src/Storage/Storage.Management/Models/PSBlobServiceProperties.cs

```diff
@@ -292,15 +295,37 @@ public CorsRule ParseCorsRule()
         /// <returns>String Array</returns>
         private static string[] ListToArray(IList<string> stringList)
         {
+            string[] stringArray = new string[stringList.Count];
```

Comment:
It looks this line will throw null reference exception if stringList is null.

Reply:
Checked in the changes by mistake. Reverted it.